test/parallel/test-stdout-close-catch.js

```diff
@@ -1,27 +1,28 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var path = require('path');
-var child_process = require('child_process');
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const child_process = require('child_process');
 
-var testScript = path.join(common.fixturesDir, 'catch-stdout-error.js');
+const testScript = path.join(common.fixturesDir, 'catch-stdout-error.js');
 
-var cmd = JSON.stringify(process.execPath) + ' ' +
-          JSON.stringify(testScript) + ' | ' +
-          JSON.stringify(process.execPath) + ' ' +
-          '-pe "process.stdin.on(\'data\' , () => process.exit(1))"';
+const cmd = JSON.stringify(process.execPath) + ' ' +
+            JSON.stringify(testScript) + ' | ' +
+            JSON.stringify(process.execPath) + ' ' +
+            '-pe "process.stdin.on(\'data\' , () => process.exit(1))"';
 
-var child = child_process.exec(cmd);
-var output = '';
-var outputExpect = { 'code': 'EPIPE',
+const child = child_process.exec(cmd);
+let output = '';
+const outputExpect = { 'code': 'EPIPE',
                      'errno': 'EPIPE',
```

Comment:
@furnox What you have as your preference there is fine. (That's my preference too but it's not universal on the project. Basically, as long as the property names are lined up, that's the main thing.)

When we get over a couple of issues that are preventing us from updating ESLint, we will have more strict linting for indentation. Thanks for bearing with us until then.

Also, while you're in here editing: it seems like the quotation marks around the property names are unnecessary. If I'm not mistaken, it could be written as:

```js
const outputExpect = {
  code: 'EPIPE',
  errno: 'EPIPE',
  syscall: 'write'
};
```

If there's a good reason for the quotation marks, then I guess keep them, but if not, perhaps take this opportunity to remove them.